packages/crayons-extended/platform-ui/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@freshworks/platform-ui",
+  "version": "1.0.0",
+  "description": "Extended crayons components",
+  "main": "dist/index.cjs.js",
+  "module": "dist/index.js",
+  "es2015": "dist/esm/index.mjs",
+  "es2017": "dist/esm/index.mjs",
+  "types": "dist/types/components.d.ts",
+  "collection": "dist/collection/collection-manifest.json",
+  "collection:main": "dist/collection/index.js",
+  "unpkg": "dist/platform-ui/platform-ui.esm.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ionic-team/stencil-component-starter.git"
+  },
+  "files": [
+    "dist/",
+    "loader/",
+    "react/"
+  ],
+  "scripts": {
+    "add:react": "npm run clean:react && npx mkdirp react",
+    "add:css": "npm run clean:css && npx mkdirp css",
+    "build": "npm run add:react  && npm run build:components && npm run build:react",
+    "build:components": "stencil build --docs",
+    "build:react": "npm run tscdisable && tsc --project tsconfig-react.json",
+    "build:mincss": "cleancss -b css/*.css css",
+    "build:scss": "node-sass --include-path node_modules --include-path ../../node_modules --output-style compressed --recursive ../../styles/global/ -o css",
+    "build:styles": "npm run build:scss && npm run build:autoprefix && npm run build:mincss ",
+    "clean:react": "rimraf react",
+    "directional-to-logical": "npx postcss './src/**/*.scss' --config ../../crayons-core/plugins/postcss-plugins/postcss.config.js --replace --no-map --verbose",
+    "generate": "stencil generate",
+    "start": "stencil build --dev --watch --serve",
+    "test": "stencil test --spec --e2e",
+    "test.watch": "stencil test --spec --e2e --watchAll",
+    "tscdisable": "node ../../../scripts/fileutil"
+  },
+  "dependencies": {
+    "@freshworks/crayons": "^3.4.2",
+    "@freshworks/crayons-i18n": "^3.1.1",
+    "@stencil/core": "2.9.0",
+    "stencil": "^0.0.5",
+    "uuid": "^8.3.2"
+  },
+  "devDependencies": {
+    "@types/jest": "^26.0.24",
```

Comment:
please add `react-output-target` as dev deps. Please check crayons-core package.json

Reply:
Added the missing dev scripts